randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2622,106 +2280,36 @@ Extra - asset_id: collision_camera_031
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Tile Group (SCREWATTACK) 2
-      Trivial
-  > Tile Group (SCREWATTACK) 3
-      Trivial
+  > Life Recharge
+      Morph Ball and Screw Attack
+  > Dock to Teleport to Burenia
+      All of the following:
+          Screw Attack
+          No Gravity Suit or Movement (Beginner)
```

Comment:
hmm, instead of No Gravity Suit, could probably make it Highly Dangerous Logic with a comment explaining picking up Gravity Suit makes this harder (hence the movement trick). Solver would probably like it better too.